src/engine/ScreenOverlays.tscn

```diff
@@ -19,6 +21,10 @@ LoadingMessagePath = NodePath("../../../ScreenOverlays/CanvasLayer/LoadingScreen
 [node name="SaveStatusOverlay" parent="CanvasLayer" instance=ExtResource( 2 )]
 visible = false
 
+[node name="QuickLoad" parent="CanvasLayer" instance=ExtResource( 5 )]
+visible = false
```

Comment:
Shouldn't this be visible? The actual dialog should be hidden by default.

Reply:
This visible bool does not make any difference. I just hid it to match the others

Reply:
I suppose that's fine then.. I wasn't the one who converted the stuff over to the new overlay manager scene.